meta-freescale-3rdparty-mbl/conf/machine/imx6ul-des0258-mbl.conf

```diff
@@ -88,3 +88,6 @@ MBL_WKS_BOOTLOADER2_SIZE_KiB = "1024"
 
 # Guess based on value for imx6ul-pico
 MBL_FLASH_ERASE_BLOCK_SIZE_KiB = "512"
+
+# Guess based on value for imx6ul-pico
+MBL_WKS_STORAGE_SIZE_MiB = "3616"
```

Comment:
@ARMmbed/team-embeddedplanet,
Can you please tell us the output of the following command in the des0258 board?
`blockdev --getsize64 /dev/mmcblk0`

Reply:
I don't have access to a des0258. I suppose I can ask @bryanodonoghue.

Reply:
Ah, I didn't notice you'd already tagged mentioned @ARMmbed/team-embeddedplanet.

Reply:
I'll resolve this conversation since we haven't received any response @ARMmbed/team-embeddedplanet.
If this value is different we can change it in another PR.